### ISSUE TITLE
Specify datetime format when reading from CSV file

### ISSUE THREAD

**Issue by alegaballo** [hosseinmoein/DataFrame#33]
Is it possible to specify the format when reading a datetime from a csv file?
It seems to me that it defaults to `YYYYMMDD` and I can't do it otherwise without modifying the source code.

**Comment by hosseinmoein** [comment]
@alegaballo,
I am not sure where you see that. The `datetime` format in the csv file is hardcoded to by `<Seconds since Epoch>.<Nanoseconds>`. You can look at the example file at `test/sample_data_dt_index.csv`.
But as you said this format is hardcoded/assumed. It is worth making it specifiable. I do not have time to do this soon but I can add it to my list. Alternatively, if you can contribute and send me a pull request, it would be great.

**Comment by alegaballo** [comment]
@hosseinmoein the default datetime format is `DT_DATE_STYLE::YYYYMMDD` [from here](https://github.com/hosseinmoein/DataFrame/blob/940a16eef7a26fafc5aed7233244bc47f53c9ae6/include/DataFrame/Utils/DateTime.h#L188), but maybe I've missed something in the code. 
I'll try to find the time to implement the format specification myself.

**Comment by hosseinmoein** [comment]
Yes, that is one of datetime constructors. It is completely unrelated to how datetimes are read/written to csv files.
